src/main/java/com/atlassian/bitbucket/jenkins/internal/jenkins/oauth/consumer/OAuthConsumerEntry.java

```diff
@@ -58,15 +58,11 @@ public static OAuthConsumerEntry getOAuthConsumerForUpdate(Consumer consumer) {
     }
 
     public String getCallbackUrl() {
-        String callback = "";
-        if (isCallbackUrlSet()) {
-            callback = consumer.getCallback().toString();
-        }
-        return callback;
+        return isCallbackUrlSet() ? consumer.getCallback().get().toString() : "";
```

Comment:
Fair enough, I'll switch it back to a conditional.
`isCallbackUrlSet` makes the `getCallback().isPresent` call. IntelliJ doesn't see it in this block, but that get call should always return a value. This method displays that callback url in a textbox, and we can't send an optional because it will print `Optional[empty]` in the front end- so it's either "" or null, and I ended up going with "".

Reply:
I think this is an example where a PR comment has caused more harm than good, I should've noticed that the value will always be present and that the whole point of this fix was to remove `Optional[blah]` from the UI. I'm happy with either the ternary or conditional, I just always get unreasonably antsy at returning empty strings :D